direct/tpm2/policy_test.go

```diff
@@ -282,25 +282,35 @@ func TestPolicyOrUpdate(t *testing.T) {
 	}
 }
 
+func getExpectedPCRDigest(t *testing.T, thetpm transport.TPM, selection tpml.PCRSelection, hashAlg tpm.AlgID) []byte {
+	pcrRead := PCRRead{
```

Comment:
nit: This is a good place to try out `t.Helper()`

Reply:
Resolved.